Examples/Python/src/Json.cpp

```diff
@@ -168,6 +168,16 @@ void addJson(Context& ctx) {
 
     mex.def("readSurfaceVectorFromJson",
             ActsExamples::JsonSurfacesReader::readVector);
+
+    py::class_<Acts::JsonDetectorElement, Acts::DetectorElementBase,
+               std::shared_ptr<Acts::JsonDetectorElement>>(
+        mex, "JsonDetectorElement")
+        .def("surface", [](Acts::JsonDetectorElement& self) {
+          return self.surface().getSharedPtr();
+        });
+
+    mex.def("readDetectorElementsFromJson",
```

Comment:
We'll probably want to refactor this to align it with the C++ namespace instead of a function at the module level. 

I also don't really understand why these functions are in the examples and not in the plugin itself.

Reply:
Yeah I agree that conceptionally it should be in the plugin since it is not a `IWriter` type object